yarn.lock

```diff
@@ -3981,6 +4039,18 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@polkadot/x-fetch@npm:^10.2.2":
+  version: 10.2.2
+  resolution: "@polkadot/x-fetch@npm:10.2.2"
+  dependencies:
+    "@babel/runtime": ^7.20.7
+    "@polkadot/x-global": 10.2.2
+    "@types/node-fetch": ^2.6.2
+    node-fetch: ^3.3.0
+  checksum: 8265dd4ebdb322e883094a5b3d4126b4443bf36a9121a0a466e1a05369366890bcf14185e23da979b61431167ae6f9500c81e4fb85ec15ce47a3c74f750a1bef
+  languageName: node
+  linkType: hard
+
```

Comment:
Something is weird where not all deps have gone up, i.e. see the ones just below, still on 10.2.1